powerhome/playbook
# [PLAY-186] Use Auto in Images Transitions

![React Kit Coverage](https://img.shields.io/badge/React%20Kit%20Coverage-12.17%25-red)
#### Screens

Could not upload the video in this PR but I uploaded it in the runway story.

#### Breaking Changes

No

#### Runway Ticket URL

[Play-186](https://nitro.powerhrg.com/runway/backlog_items/PLAY-186)

#### How to test this

In dev tools click the network tab and there will be a `Disable cache` checkbox. When checked images will not be cached and the transitions will work as normal, also the class `ls-is-cached` will not appear on the `img` element. When unchecked images will be checked and the images will immediately appear without the transitions, also the class `ls-is-cached` will appear on the `img` element.

#### Checklist:
